packages/web/components/Form/HelpModal.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import Modal from 'react-overlays/Modal';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+	position: fixed;
+	z-index: 1000;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	background-color: ${({ theme }) => theme.colors.black};
+	opacity: 0.7;
+`;
+
+const StyledModal = styled(Modal)`
+	position: fixed;
+	top: 40%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	width: 630px;
+	max-width: 94%;
+	z-index: 1000;
+	background-color: ${({ theme }) => theme.colors.white};
+	box-shadow: 0 0 20px 0 ${({ theme }) => theme.colors.black};
+	padding: 40px 32px;
+	border-radius: 0.5rem;
+	text-align: left;
+	color: ${({ theme }) => theme.colors.lightGray};
+	max-height: 60%;
+	overflow-y: auto;
+`;
```

Comment:
Veja se não é mais interessante chamar o tema na raiz dos estilos, fica mais limpo:

```suggestion
const StyledModal = styled(Modal)`
	${({ theme }) => css`
		position: fixed;
		top: 40%;
		left: 50%;
		transform: translate(-50%, -50%);
		width: 630px;
		max-width: 94%;
		z-index: 1000;
		background-color: ${theme.colors.white};
		box-shadow: 0 0 20px 0 ${theme.colors.black};
		padding: 40px 32px;
		border-radius: 0.5rem;
		text-align: left;
		color: ${theme.colors.lightGray};
		max-height: 60%;
		overflow-y: auto;
	`}
`;

```

Reply:
Só me metendo para deixar meus 5 cents aqui :smiley: 
Acho que essa 2ª estrutura fica melhor mesmo, @calaca . Inicialmente, nós estávamos fazendo como você fez, chamando sempre o `theme` em cada propriedade. Depois percebemos que ficava verboso, mas aí não valeria a pena sair alterando tudo...então, à medida que mexíamos em algum componente ou que criávamos um novo, fazíamos já com esse novo estilo. Por isso que temos das 2 formas no código.

Um exemplo desse 2 modo: https://github.com/ufersa/plataforma-sabia/blob/70c14254498977b1f7d9a500cdf89330d1bf51e2/packages/web/components/MainSearch/styles.js#L20

Reply:
@luizeboli @lcnogueira concordo totalmente com os pontos de vcs levantaram. Obrigada pelas sugestões! :slightly_smiling_face: